tests/erlang_tests/test_timeout_not_integer.erl

```diff
@@ -0,0 +1,67 @@
+%
+% This file is part of AtomVM.
+%
+% Copyright 2022 Paul Guyot <pguyot@kallisys.net>
+%
+% Licensed under the Apache License, Version 2.0 (the "License");
+% you may not use this file except in compliance with the License.
+% You may obtain a copy of the License at
+%
+%    http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS,
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+% See the License for the specific language governing permissions and
+% limitations under the License.
+%
+% SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+%
+
+-module(test_timeout_not_integer).
+
+-export([start/0]).
+
+start() ->
+    test_infinity() +
```

Comment:
Nit:  maybe one line?  (I don't care, really, not worth a full rebuild, unless you are doing other stuff)

Reply:
I indented this differently and then erlfmt changed it and I agree this is quite ugly.